docs-ref-services/latest/msal-python-overview.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Microsoft Authentication Library (MSAL) for Python overview
+titleSuffix: Microsoft Authentication Library (MSAL)
+description: Learn how to integrate MSAL for Python with the Microsoft identity platform
+services: active-directory
+author: mmacy
+manager: CelesteDG
+
+ms.service: active-directory
+ms.subservice: develop
+ms.topic: conceptual
+ms.workload: identity
+ms.date: 01/19/2021
+ms.author: marsma
+ms.custom: marsma
+#Customer intent: As an application developer, I want to know how to write a desktop app that calls web APIs by using the Microsoft identity platform.
+---
+# Microsoft Authentication Library (MSAL) for Python
+
+
+| `dev` branch | Reference Docs | # of Downloads
```

Comment:
There are a few things from the README that we can drop for docs.microsoft.com, like this build status table. I can work with you to get the format right.